apigee_api_catalog.links.task.yml

```diff
@@ -19,3 +19,8 @@ entity.apidoc.delete_form:
   base_route:  entity.apidoc.canonical
   title: 'Delete'
   weight: 10
+
+apidoc.admin:
```

Comment:
Should we weight this route to show up at the end of the tab list?  It seems like our route should show up after standard Drupal tabs.

Reply:
I added the weight and this is how it looks: 
<img width="701" alt="image" src="https://user-images.githubusercontent.com/159693/58041336-ba6bac80-7aec-11e9-9a94-1ec5d91bf3bc.png">